modules/dr-restoring-cluster-state.adoc

```diff
@@ -554,6 +554,8 @@ etcd-ip-10-0-154-194.ec2.internal                2/2     Running     0
 etcd-ip-10-0-173-171.ec2.internal                2/2     Running     0          9h
 ----
 
+To ensure that all workloads return to normal operation following a recovery procedure, you should restart each pod that stores Kubernetes API information. This includes {product-title} components such as routers, operators, and third-party components.
```

Comment:
SGTM as a general pointer, but deferring to @hexfusion if we don't want to be more specific/prescriptive about including steps to reboot worker machines. Or if that's too disruptive then we can leave it up to the users to decide how best to reboot individual components.

Reply:
Thanks @hasbro17 
@hexfusion  Your thoughts on Haseeb's input?

Reply:
Hi @hexfusion I requested a QE review but can make any changes that you have and have it re-reviewed. Thanks!